java_tools/configuration_definition_base/build.gradle

```diff
@@ -5,6 +5,7 @@ defaultTasks 'shadowJar'
 dependencies {
     api project(':inifile')
     api project(':enum_to_string')
+    api project(':enum_generator')
```

Comment:
no reason for 'base' to access enum_generator? see https://github.com/rusefi/rusefi/commit/c83912a6fa7a97af0622318be7694aca3110c515

@awpe please rebase and squash. Forgive me for weird merge approach, small brain here need your help dividing